client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/usePersonalInfoTab.ts

```diff
@@ -11,11 +11,11 @@ const usePersonalInfoTab = (parameters: usePersoanlInfoTabParameters): usePerson
 
     const epidemiologyNumber = useSelector<StoreStateType, number>(state => state.investigation.epidemiologyNumber);
 
-    const {setOccupations, setInsuranceCompanies, personalInfoStateContext, 
+    const {setInsuranceCompanies, personalInfoStateContext, occupationsStateContext,
         setSubOccupations, setSubOccupationName, setCityName, setStreetName, setStreets} = parameters;
 
     const fetchPersonalInfo = () => {
-        axios.get('/personalDetails/occupations').then((res: any) => res && res.data && res.data.data && setOccupations(res.data.data.allOccupations.nodes.map((node: any) => node.displayName)));
+        axios.get('/personalDetails/occupations').then((res: any) => occupationsStateContext.occupations = res?.data?.data?.allOccupations?.nodes?.map((node: any) => node.displayName));
```

Comment:
You should load it when first mounting the whole form and not the tab

Reply:
I didn't change it but after elad's team refactor data is loaded in tabs anyway